models/intel/text-to-speech-en-multi-speaker-0001/README.md

```diff
@@ -0,0 +1,128 @@
+# text-to-speech-en-multi-speaker-0001 (composite)
```

Comment:
```suggestion
# text-to-speech-en-multi-0001 (composite)
```
suggest to have unified base part of the name for composite models, like we have for all other composite models

Reply:
Done